<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lwt_contents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>39</y>
      <width>921</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>861</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>物品增加</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_id">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_name">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>种类</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>物品ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_position">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>位置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_date_expiration">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_supplier">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>供应商</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>有效日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>入库时间</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_price">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_type">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>工具</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>仪器</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>消耗品</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmb_state">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>库房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>借出</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>损坏</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView_add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>821</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>250</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_date_enter">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_owner">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>拥有者</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>861</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>物品总览</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>440</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>390</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>物品ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_id_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>390</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_name_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>440</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>490</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sort_state">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>根据状态分类</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sort_type">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>根据种类分类</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_expire_hint">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>290</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>即将过期物品</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>751</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_overview">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>60</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>物品总览</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldt_days">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>250</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>天</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>再过</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>861</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>物品状态</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>891</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>物品状态</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>物品ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_search_state">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>物品状态</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_id_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_state">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>871</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>物品借出</string>
      </property>
      <widget class="QPushButton" name="btn_borrow">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>借出</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>借出人</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_name_state">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>物品归还</string>
      </property>
      <widget class="QPushButton" name="btn_return">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>归还</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>420</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>物品损坏</string>
      </property>
      <widget class="QPushButton" name="btn_damage">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>损坏</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_reason">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>损坏原因</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>861</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>软件简介</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>71</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;通过物品名及物品编号查找物品 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;给物品编码然后扫描编码显示物品。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;点击物品显示状态（借出，库房，以及损坏） &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  借出：显示物品由xxx于xx年xx月xx日xx时借出以及借出用途。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  归还：归还日期归还人 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  损坏：当时日期损坏人损坏原因 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;库房；显示物品在库房的位置。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;   借出：借出人，获取当时的时间日期及借出用途。 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;   损坏：当时日期损坏人损坏原因 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>861</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>基于RFID的库房管理系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
